src/types/sitedashboard.ts

```diff
@@ -0,0 +1,30 @@
+export enum ReviewRequestStatus {
+  PENDING = "PENDING",
+  APPROVED = "APPROVED",
+}
+
+export type CollaboratorsStats = {
+  total: number
+  inactive: number
+}
+
+export type SiteDashboardReviewRequest = {
+  id: number
+  title: string
+  description: string
+  author: string
+  status: ReviewRequestStatus
+  changedFiles: number
+  newComments: number
+  firstView: boolean
+  createdAt: number // Unix timestamp
+}
+
+export type SiteDashboardInfo = {
+  savedAt: number // Unix timestamp
```

Comment:
is this a todo conversion to a unix timestamp? If it is not meant to be of type `datetype` may i get your help to comment the expected shape of the timestamp, eg, DDMMYYYY?

Reply:
Yup! This is meant to be the unix timestamp, and the conversion is done by @seaerchin in #1071 under `getDateTimeFromUnixTime()`.